tools/Deployment/gulpfile.js

```diff
@@ -10,6 +10,8 @@ let del = require("del");
 let glob = require("glob");
 let gulp = require("gulp");
 let nconf = require("nconf");
+let format = require("string-format");
+format.extend(String.prototype, {})
```

Comment:
Use this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
`Template_literals` requires to prepare the variables first, but I want to set this in the config.